xbmc/cores/RetroPlayer/rendering/RPRenderManager.cpp

```diff
@@ -177,7 +182,13 @@ void CRPRenderManager::FrameMove()
 
     if (m_state == RENDER_STATE::CONFIGURING)
     {
+      CLog::Log(LOGDEBUG, "RetroPlayer[RENDER]: Configuring renderer(s)");
+
+      for (auto &renderer : m_renderers)
+        renderer->Configure(m_format, m_width, m_height);
```

Comment:
Renderers are created on demand, and configured on creation, so this shouldn't be necessary. Is `m_renderers` even nonempty at this point?

Reply:
This is critical. It is for re-configuring existing renderers on resolution change. I will add a comment to clarify.

Reply:
I see this now. Thanks.